practice/src/css/bases/variable.css

```diff
@@ -0,0 +1,36 @@
+:root {
+  --primary-font: "Barlow";
+
+  --font-weight-semibold: 600;
+  --font-weight-medium: 500;
+  --font-weight-regular: 400;
+
+  --primary-text-color: #fff;
```

Comment:
pls consistent HEX code for 6 characters

Reply:
ignore this comment

Reply:
@JennyDoJD You can refer this article https://www.freecodecamp.org/news/how-hex-code-colors-work-how-to-choose-colors-without-a-color-picker/

```
you can [use shorthand to optimize CSS delivery](https://hostingcanada.org/css-optimization/), encoding your colors using abbreviations can improve the performance of your web pages. This might not be visible when working with small pages, but can have a significant impact as your projects get more complex.
```

Please by pass the Canh's comment